Sources/MapboxNavigation/MapView.swift

```diff
@@ -198,4 +198,27 @@ extension MapView {
             return VectorSource.isMapboxStreets(identifiers)
         }
     }
+    
+    /**
+     Attempts to localize road labels into the local language and other labels into the given locale.
+     */
+    func localizeLabels(into locale: Locale) {
+        guard !ResourceOptionsManager.hasChinaBaseURL else { return }
+        
+        let style = mapboxMap.style
+        guard let mapboxStreetsSource = streetsSources().first else { return }
+        
+        let streetsSourceTilesetIdentifiers = tileSetIdentifiers(mapboxStreetsSource.id)
+        let roadLabelSourceLayerIdentifier = streetsSourceTilesetIdentifiers.compactMap { VectorSource.roadLabelLayerIdentifiersByTileSetIdentifier[$0]
+        }.first
+        
+        let localizableLayerIdentifiers = style.allLayerIdentifiers.filter { $0.type == .symbol }
+            // We only know how to localize layers backed by the Mapbox Streets source.
+            .filter { style.layerProperty(for: $0.id, property: "source") as? String == mapboxStreetsSource.id }
+            // Road labels should match road signage, so they should not be localized.
+            // TODO: Actively delocalize road labels into the “name” property: https://github.com/mapbox/mapbox-maps-ios/issues/653
```

Comment:
Road labels are untouched and remain in English if the original style had them in English, as does the default Mapbox Navigation Day/Night v1 style. This is due to a missing special case upstream: mapbox/mapbox-maps-ios#653. It’s a regression from v1._x_.